### PR TITLE
VALIDATE $STICKY

### PR DESCRIPTION
**Pull request by pepesolanastrict**
# Validate $STICKY (https://solscan.io/token/8tgtGXhYRGKtaF4CPeVzSQpM3cUXBZ8P4ZW4hEhyLWZZ)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/STICKMANPRIDE/status/1796609659727483140?t=66bUHQOQjYhFbm0QL50CuA&s=19
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/YOUR_ID_HERE
Applied
## Validation (Please check off boxes):
- [X] The metadata provided in the PR matches what is on-chain (Mandatory)
- [X] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [X] My change matches the format in the file (no spaces between fields).
- [X] My token is already live and trading on Jupiter.
- [X] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [X] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by Guarmarketing — COMMENTED** [review]
good to be on strict list

**Comment by YoungStuhl** [comment]
Thanks for pulling it :)

**Review by YoungStuhl — COMMENTED** [review]
very good for strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1685,3 +1685,4 @@ Wine,WINE,DoVRzHXXicZucnuskdMg7DePhBHFkhvvhta5nf1deFoQ,6,https://ipfs.io/ipfs/Qm
 HOPE,HOPE,9Zhvjcce3kHbYQGojybM9GbtE76rvoMJM36egCcmPs2s,9,https://ipfs.io/ipfs/QmfJE2zGbSrFzopqis8B9AAfKRZHLYt6GDjcB368zBmvSB,true
 RoaringPepe,RPEPE,8jnGePqBBq1KQfPvqA7zAZySiFTyy8tv2zA8u6nHhsdo,6,https://ipfs.io/ipfs/QmbC7tJ3EieLuAm4GeuUWsqhyCdidASrarJ6hZZTQDzWdd,true
 gomu gator,gomu,Bx74hpFiaiBbSonrjyqxjGfAA7gRxM2CeKiy31uN6biR,6,https://ipfs.io/ipfs/QmSHQy78zSk1eaG8cFGiLEeoyWAEB2ro3S76WpJZ1FfjKW,true
+stickmanpride,$STICKY,8tgtGXhYRGKtaF4CPeVzSQpM3cUXBZ8P4ZW4hEhyLWZZ,6,https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f63662d697066732e636f6d2f697066732f516d5643735574704e3171616f784a77656269466a6a364534535555446362427962756174327431516f50375342,true
```
Comment (pepesolanastrict) [inline validated-tokens.csv:1688]: Add to Strict List

**Comment by tomsitoms** [comment]
Strict list

**Comment by DirtyMike617** [comment]
Please review asap we're ready and willing

**Comment by Jput50** [comment]
Please review asap

**Comment by thirdstretch** [comment]
Ready for review.

**Comment by Rich-Pappy** [comment]
Please review ASAP

**Comment by jmesparza06** [comment]
Please review ASAP.

**Comment by Jymfletzfur** [comment]
insufficient volume

**Pull request closed without merging**